docs/src/main/sphinx/functions/binary.rst

```diff
@@ -172,7 +172,17 @@ Hashing functions
 
 .. function:: murmur3(binary) -> varbinary
 
-    Computes the 128-bit murmur3 hash of ``binary``.
+    Computes the 128-bit murmur3 hash of ``binary``::
```

Comment:
use MurmurHash3 and link to https://en.wikipedia.org/wiki/MurmurHash

```
Computes the 128-bit `MurmurHash3 <https://en.wikipedia.org/wiki/MurmurHash>`_ value of 
```

Reply:
done, Thank you for your help.